Upstream: in master, without NEWS and ChangeLog changes
From: John Stanley <jpsinthemix@verizon.net>
Date: Fri, 9 Sep 2011 02:57:23 +0000 (-0400)
Subject: Add missing definition of LOCPATH for bug-regex32 run
X-Git-Url: http://sourceware.org/git/?p=glibc.git;a=commitdiff_plain;h=b0727fd83f0f27c73f4c664feb6891acc328d160

Add missing definition of LOCPATH for bug-regex32 run
---

diff --git a/posix/Makefile b/posix/Makefile
index 499d53d..b200342 100644
--- a/posix/Makefile
+++ b/posix/Makefile
@@ -208,6 +208,7 @@ bug-regex23-ENV = LOCPATH=$(common-objpfx)localedata
 bug-regex25-ENV = LOCPATH=$(common-objpfx)localedata
 bug-regex26-ENV = LOCPATH=$(common-objpfx)localedata
 bug-regex30-ENV = LOCPATH=$(common-objpfx)localedata
+bug-regex32-ENV = LOCPATH=$(common-objpfx)localedata
 tst-rxspencer-ARGS = --utf8 rxspencer/tests
 tst-rxspencer-ENV = LOCPATH=$(common-objpfx)localedata
 tst-pcre-ARGS = PCRE.tests
